### PR TITLE
Update fdm_pl.po

### PR DESCRIPTION
**Pull request by BestiaPL**

### CODE REVIEW

File: main/fdm_pl.po [hunk 1]
```diff
@@ -1499,9 +1499,9 @@ msgstr "Brak połączenia z Internetem"
 msgctxt "MainStatusBar|"
 msgid "%n downloads selected."
 msgid_plural "%n downloads selected."
-msgstr[0] ""
-msgstr[1] ""
-msgstr[2] ""
+msgstr[0] "Jedno pobieranie zaznaczone"
```
Comment (FreeDownloadManagerTeam) [inline main/fdm_pl.po:1502]:
```suggestion
msgstr[0] "%n pobieranie zaznaczone."
```

File: main/fdm_pl.po [hunk 2]
```diff
@@ -1499,9 +1499,9 @@ msgstr "Brak połączenia z Internetem"
...
+msgstr[1] "%n pobierania zaznaczone"
```
Comment (FreeDownloadManagerTeam) [inline main/fdm_pl.po:1503]:
```suggestion
msgstr[1] "%n pobierania zaznaczone."
```

File: main/fdm_pl.po [hunk 3]
```diff
@@ -1499,9 +1499,9 @@ msgstr "Brak połączenia z Internetem"
...
+msgstr[2] "%n pobierań zaznaczonych"
```
Comment (FreeDownloadManagerTeam) [inline main/fdm_pl.po:1504]:
```suggestion
msgstr[2] "%n pobierań zaznaczonych."
```

File: main/fdm_pl.po (hunk 1 above)
Comment (BestiaPL) [reply main/fdm_pl.po:1502]: > 

Right, but if this is "one" in Polish we are writting "Jedno /something/", not "1 /something/".
Thanks for dots at the end of strings.

**Pull request merged**